css/components/input.css

```diff
@@ -0,0 +1,45 @@
+@import url(../base/reset.css);
+@import url(../helpers/typography.css);
+@import url(../helpers/color.css);
+
+
+
+.input-group, .textarea-group{
+    display: flex;
+    flex-direction: column;
+    margin: 1rem;
+}
+
+.input-outline, .textarea-outline{
+    margin-top: 3px;
```

Comment:
Suggestion: can use shorthand property for margin

Reply:
Done 👍🏻